gulpfile.js

```diff
@@ -0,0 +1,13 @@
+var gulp = require('gulp');
+var replace = require('gulp-replace');
+var pjson = require('./package.json');
+
+gulp.task('propagate-version', function() {
+  gulp.src(['./bower.json'])
+    .pipe(replace(/"version": "\d+\.\d+\.\d+"/, '"version": "' + pjson.version + '"'))
+    .pipe(gulp.dest('./'));
+
+  gulp.src(['./juicy-filedrop.html'])
```

Comment:
But they don't have to, it just require a little bit more sophisticated regexp. We can pick one from grunt-bump: https://github.com/vojtajina/grunt-bump/blob/ed9e086f7a8d76891c4628db0977cb3998bdfc0c/tasks/bump.js#L49
As I would say `/"version": "\d+\.\d+\.\d+"/` is not enough anyways (`"version":"1.2.3"` will not pass, needless to mention pre-releases)